include/envoy/router/router.h

```diff
@@ -786,6 +786,8 @@ class Config {
    * @return const std::string the RouteConfiguration name.
    */
   virtual const std::string& name() const PURE;
+
+  virtual bool usesVhds() const PURE;
```

Comment:
How is this used? I don't see any call sites. Is this for later PRs?

Reply:
Yes, it's used in on-demand VHDS. I can remove it here if you prefer.

Reply:
Yeah, could be good to punt to that.